floor/floor/controller/controller.py

```diff
@@ -162,14 +162,15 @@ def generate_frame(self):
         composited_leds = [(0, 0, 0)] * 64
         last_leds = None
         for layer in self._iter_enabled_layers():
+            alpha = layer.get_alpha()
             current_leds = layer.render(context, leds=last_leds)
             if not current_leds:
                 continue
             for idx, current_pixel in enumerate(current_leds):
                 current_pixel = normalize_pixel(current_pixel)
                 if last_leds:
                     last_pixel = last_leds[idx]
-                    composited_leds[idx] = blend_pixel_copy(last_pixel, current_pixel)
+                    composited_leds[idx] = alpha_blend(current_pixel, last_pixel, alpha)
```

Comment:
So I think this should be something like:
```python
last_pixel = composited_leds[idx]
composited_leds[idx] = alpha_blend(current_pixel, last_pixel, alpha)
```
because when there are two overlays, the 2nd overlay clobbers contribution from the base layer, e.g.:
1) base = (100, 100, 100) >> composite = (100, 100, 100)
2) overlay1 = (200, 0, 0) @ alpha=50 acts on base >> composite = (150, 50, 50)
3) overlay2 = (0, 200, 0) @ alpha=50 acts on unaltered overlay1 >> composite = (100, 100, 0)

My expectation is that #3 would look something like:

3) overlay2 = (0, 200, 0) @ alpha=50 acts on current composite >> composite = (75, 125, 25)

The only problem is that layer 1 & 2 end up being 25% alpha and 25% alpha.  Maybe there is a smarter way of doing this?  Or maybe that's good enough and people can play with the sliders to get whatever effect they want.

Reply:
> when there are two overlays, the 2nd overlay clobbers contribution from the base layer [..]

Ah, good catch! Fixed this in d8aaded, which also adds a test for this in `controller_test.py`